content/applications/hr/time_off.rst

```diff
@@ -106,21 +78,76 @@ Allocation requests section
 - :guilabel:`Employee Requests`: select :guilabel:`Extra Days Requests Allowed` if the employee is
   able to request more time off than was allocated.
 
-  .. example::
-     For example, if ten (10) days are allocated to the employee for this particular type of time
-     off, and this option is enabled, the employee may submit a request for more than ten (10) days.
-
   If employees should **not** be able to make requests for more time off than what was allocated,
   select the :guilabel:`Not Allowed` option.
 
+  .. example::
+     Ten (10) days are allocated to the employee for this particular type of time off, and the
```

Comment:
I'm not really understanding why we need the numerical value of the days in parentheses, but if you have a reason for doing so, feel free to ignore this suggestion
```suggestion
     Ten days are allocated to the employee for this particular type of time off, and the
```

Reply:
Also, what is "particular type of time off" referring to? Is this Employee Requests? if so, maybe we should specify that so there's absolutely no confusion